### PR TITLE
11.0 updates

### PR DESCRIPTION
**Pull request by Solanya**
The list of changes is as follows (unless I'm forgetting something):
- Updated the UI elements affected by the big base TRP UI rework.
- Fixed new API calls (GetMouseFoci)
- Fixed the cutscene number icons since Blizzard removed the template
- Added trade/inspection commands to the right-click menus
- Reenabled local sounds and musics in instances for players in the same group/raid.
- Updated all the dropdowns with the new template and fixed their sizes and positions
- Fixed the size/alignment of UI elements in a few places
- Automatically close the item/campaign creation popups when clicking on the Quick Import button

### CODE REVIEW

File: totalRP3_Extended/Quest/QuestObjective.lua [hunk 1]
```diff
@@ -15,6 +16,10 @@ local ACTION_FRAMES = {
 };
 
 local function display()
+	if not colorQuestYellow then
+		colorQuestYellow = TRP3_API.CreateColorFromBytes(190, 155, 0);
+	end
+
```
Comment (Meorawr) [inline totalRP3_Extended/Quest/QuestObjective.lua:22]: Any reason this couldn't just be initialized in the script body?

**Review by Meorawr — APPROVED** [review]
Haven't found any glaringly obvious issues on a quick fist pass, noting that 90% of the diff is just dropdown template renames and position adjustments anyway.

File: totalRP3_Extended/UnitPopups.lua [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        "PLAYER",
+        "RAID",
+        "RAID_PLAYER",
+    };
+
+    for _, unitType in pairs(unitTypesAllowed) do
+        tinsert(TRP3_UnitPopupsModule.MenuEntries[unitType], "ExtendedOpenExchange");
+        tinsert(TRP3_UnitPopupsModule.MenuEntries[unitType], "ExtendedCharacterInspection");
+    end
+end
```
Comment (Raenore) [inline totalRP3_Extended/UnitPopups.lua:95]:
```suggestion
end

```
Missing EOL newline.

File: totalRP3_Extended/Widgets.lua [hunk 3]
```diff
@@ -40,3 +40,13 @@ TRP3_BACKDROP_MIXED_TUTORIAL_TOOLTIP_418_24_5555 = {
...
+
+TRP3_BACKDROP_MIXED_BANK_TOOLTIP_100_16_4222 = {
+    bgFile   = "Interface\\BankFrame\\Bank-Background",
+    edgeFile = "Interface\\Tooltips\\UI-Tooltip-Border",
+    tile     = true,
+    tileEdge = true,
+    tileSize = 100,
+    edgeSize = 16,
+    insets   = { left = 4, right = 2, top = 2, bottom = 2 },
+};
```
Comment (Raenore) [inline totalRP3_Extended/Widgets.lua:52]:
```suggestion
};

```
Another one.

**Review by Raenore — APPROVED** [review]
Checked everything, unfortunately I obviously didn't run it in-game.

But no glaring errors at least, so if something breaks you're only like 95% responsible or something!

**Pull request merged**